terraform/bastion/root_data.tf

```diff
@@ -0,0 +1,33 @@
+data "aws_ssm_parameter" "cost_centre" {
```

Comment:
Is it worth passing in "consignmentapi" as a var, so its a bit more generic? Not sure this is practical here though.

Reply:
Yeah, can do actually, that way if we name the databases sensibly we could spin it up with access to a different services database. We probably won't but it's easy enough to change so I've set it as a variable and put consignmentapi as the default.